### PR TITLE
SWC-7091: use newerThanTimestamp in TraceAgentsRequest, do not revers…

### PR DESCRIPTION
**Pull request by jay-hodgson**
…e the trace events (they are added in ascending order), and rework mock responses to align with expectations (return a new event on some calls, but an empty array for most of the status checks)

### CODE REVIEW

**Comment by jay-hodgson** [comment]
With simulated delay in processing:
chrome_NzxyNzcNus

File: packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx [hunk 1]
```diff
@@ -279,46 +302,40 @@ export const SynapseChat: React.FunctionComponent<SynapseChatProps> = ({
...
-                            }
+                  <Box sx={{ position: 'relative', pt: '35px' }}>
+                    <TransitionGroup>
+                      {/* The key is based on the current text so when the text changes, the Fade component will remount */}
+                      <Fade key={latestTraceEventMessage} timeout={500}>
+                        <Tooltip
+                          placement="bottom"
+                          title={
+                            <div style={{ textAlign: 'center' }}>
+                              <Typography>Initializing...</Typography>
```
Comment (nickgros) [inline packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx:313]: Probably only show this node if `allTraceEvents` is empty

**Review by nickgros — APPROVED** [review]

File: packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx (hunk 1 above)
Comment (jay-hodgson) [reply packages/synapse-react-client/src/components/SynapseChat/SynapseChat.tsx]: nice catch

**Comment by jay-hodgson** [comment]
Ignoring SageAccountWeb test failure in ResetTwoFactorAuth.test.tsx (ReferenceError: window is not defined)

**Pull request merged**